### PR TITLE
feat(chat-e2e): unpublish folder tests

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

unpublish folder tests

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2754.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2754.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2754.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2754.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat-e2e/src/assertions/publishFolderEntityAssertion.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { FolderAssertion } from '@/src/assertions/folderAssertion';
+import { PublishingExpectedMessages, TreeEntity } from '@/src/testData';
+import { PublishFolder } from '@/src/ui/webElements/entityTree';
+
+export class PublishFolderEntityAssertion<
```
Comment (nartovm) [inline apps/chat-e2e/src/assertions/publishFolderEntityAssertion.ts]: Is that possible for us to have folders with version not in the publishing flow only? Probably we should move these asertions to the folderAssertion class?

Comment (irinakartun) [reply apps/chat-e2e/src/assertions/publishFolderEntityAssertion.ts]: not, version is defined only for publishing that's why I created separated assertion class

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2754.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by nartovm — APPROVED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2754.nightly-test.deltixhub.io
>E2E tests status: success

**Review by denys-kolomiitsev — APPROVED** [review]

**Pull request merged**